Makefile

```diff
@@ -122,7 +122,7 @@ PROC_CMD = --procs ${PROCS}
 
 .PHONY: test
 test: manifests generate fmt vet envtest ginkgo ## Run tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) -v debug --bin-dir $(LOCALBIN) use $(ENVTEST_K8S_VERSION) -p path)" OPERATOR_TEMPLATES="$(PWD)/templates" $(GINKGO) --trace --cover --coverpkg=../../pkg/ovndbcluster,../../pkg/ovnnorthd,../../pkg/ovncontroller,../../controllers,../../api/v1beta1 --coverprofile cover.out --covermode=atomic --randomize-all ${PROC_CMD} $(GINKGO_ARGS) ./tests/...
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) -v debug --bin-dir $(LOCALBIN) use $(ENVTEST_K8S_VERSION) -p path)" OPERATOR_TEMPLATES="$(shell pwd)/templates" $(GINKGO) --trace --cover --coverpkg=../../pkg/ovndbcluster,../../pkg/ovnnorthd,../../pkg/ovncontroller,../../controllers,../../api/v1beta1 --coverprofile cover.out --covermode=atomic --randomize-all ${PROC_CMD} $(GINKGO_ARGS) ./tests/...
```

Comment:
why this change?

Reply:
pushed a seperate commit for this , reason is To allow running tests from different directory. as how ci framework execute these target from different directory, this was required to fix content provider job, neutron operator already had this fix applied